docs/build/voting-dapp-tutorial.md

```diff
@@ -0,0 +1,428 @@
+---
+title: dApp Tutorial
+description: Build a complete dApp on Kadena
+---
+
+import PageRef from '@components/PageRef'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Complete dApp Tutorial
+
+Build a complete on-chain voting dApp
+
+---
+
+## Overview
+
+In this tutorial you will build a complete dApp, smart contracts and frontend that allows users to submit a vote. You will also deploy your own gas station that will be used to pay for the gas needed to interact with your dApp.
```

Comment:
Lets try and add a few paragraphs to motivate why a decentralized ledger is a good fit for this sort of application / what a smart contract is able to do that would otherwise not be possible on web2-style tech

Reply:
Added a paragraph here, feedback is welcome.

Reply:
I also thought "submit a vote" sounds pretty lame, thanks for the suggestion